_fossfund/9-sharplab.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: SharpLab
-logo: 
+logo: sharplab.svg
```

Comment:
I think the reason we had this as blank is that we could not find a logo, I took another look for one and still came up empty. Its ok if this field is null